hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/JetSqlSerializerHook.java

```diff
@@ -72,9 +144,92 @@ public class JetSqlSerializerHook implements DataSerializerHook {
+    public static final int INDEX_FILTER_RANGE = 29;
+    public static final int INDEX_FILTER_IN = 30;
+    public static final int EXPRESSION_TO_CHAR = 31;
+    public static final int EXPRESSION_COLUMN = 32;
+    public static final int EXPRESSION_IS_NULL = 33;
+    public static final int EXPRESSION_CONSTANT = 34;
+    public static final int EXPRESSION_PARAMETER = 35;
+    public static final int EXPRESSION_CAST = 36;
+    public static final int EXPRESSION_DIVIDE = 37;
+    public static final int EXPRESSION_MINUS = 38;
+    public static final int EXPRESSION_MULTIPLY = 39;
+    public static final int EXPRESSION_PLUS = 40;
+    public static final int EXPRESSION_UNARY_MINUS = 41;
+    public static final int EXPRESSION_AND = 42;
+    public static final int EXPRESSION_OR = 43;
+    public static final int EXPRESSION_NOT = 44;
+    public static final int EXPRESSION_COMPARISON = 45;
+    public static final int EXPRESSION_IS_TRUE = 46;
+    public static final int EXPRESSION_IS_NOT_TRUE = 47;
+    public static final int EXPRESSION_IS_FALSE = 48;
+    public static final int EXPRESSION_IS_NOT_FALSE = 49;
+    public static final int EXPRESSION_IS_NOT_NULL = 50;
+
+    public static final int EXPRESSION_ABS = 51;
+    public static final int EXPRESSION_SIGN = 52;
+    public static final int EXPRESSION_RAND = 53;
+    public static final int EXPRESSION_DOUBLE = 54;
+    public static final int EXPRESSION_FLOOR_CEIL = 55;
+    public static final int EXPRESSION_ROUND_TRUNCATE = 56;
+
+    //region String expressions IDs
+    public static final int EXPRESSION_ASCII = 57;
+    public static final int EXPRESSION_CHAR_LENGTH = 58;
+    public static final int EXPRESSION_INITCAP = 59;
+    public static final int EXPRESSION_LOWER = 60;
+    public static final int EXPRESSION_UPPER = 61;
+    public static final int EXPRESSION_CONCAT = 62;
+    public static final int EXPRESSION_LIKE = 63;
+    public static final int EXPRESSION_SUBSTRING = 64;
+    public static final int EXPRESSION_TRIM = 65;
+    public static final int EXPRESSION_REMAINDER = 66;
+    public static final int EXPRESSION_CONCAT_WS = 67;
+    public static final int EXPRESSION_REPLACE = 68;
+    public static final int EXPRESSION_POSITION = 69;
+    public static final int EXPRESSION_CASE = 70;
+    public static final int EXPRESSION_EXTRACT = 71;
+    //endregion
+
+    public static final int EXPRESSION_DOUBLE_DOUBLE = 72;
+    public static final int EXPRESSION_TO_TIMESTAMP_TZ = 73;
+    public static final int EXPRESSION_TO_EPOCH_MILLIS = 74;
+    public static final int SARG_EXPRESSION = 75;
+    public static final int SEARCH_PREDICATE = 76;
+    public static final int EXPRESSION_FIELD_ACCESS = 77;
+    public static final int EXPRESSION_ROW = 78;
+
+    public static final int MAP_FETCH_INDEX_OPERATION = 80;
+    public static final int INDEX_ITERATION_POINTER = 81;
+    public static final int MAP_FETCH_INDEX_OPERATION_RESULT = 82;
```

Comment:
I see that these classes are in the hazelcast module, not hazelcast-sql. And, they are defined in the MapDataSerializerHook as well. Why are we re-defining them here?